core/resource/testing/resource.go

```diff
@@ -34,9 +32,7 @@ func NewResource(c *gc.C, stub *testing.Stub, name, applicationID, content strin
 // content. The origin is set to "upload" (via resource created by  NewCharmDockerResource).
 // A reader is also returned which contains the content.
 func NewDockerResource(c *gc.C, stub *testing.Stub, name, applicationID, content string) resource.Opened {
-	username := "a-user"
 	return resource.Opened{
-		Resource:   newResource(c, name, applicationID, username, content, NewCharmDockerResource),
```

Comment:
**suggestion:** `NewResource` and `NewDockerResource` are not a resource anymore if you do this change.
Maybe use `resource.Content` and call it newContent. Or remove it. 

Or Don't change it and move it in the test (doesn't seems to be called outside tests, looks like those are convenient function for Arrange tests.

Reply:
Decided not to change this file at all.